Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3993,7 +3993,7 @@ ManageWebKitProcessesAsExtensions:
   exposed: [ WebKit ]
   defaultValue:
     WebKit:
-      default: false
+      default: true
```

Comment:
You need to limit the default to just Cocoa-based builds, otherwise we break the other ports.

```
    WebKit:
      "PLATFORM(COCOA)": true
      default: false
```

Reply:
Fixed!

Thanks for reviewing!